tests/console/valgrind.pm

```diff
@@ -44,7 +44,7 @@ sub run {
     assert_script_run 'cd /var/tmp';
     assert_script_run 'curl -v -o valgrind-test.c ' . data_url('valgrind/valgrind-test.c');
     assert_script_run 'curl -v -o valgrind-test.sh ' . data_url('valgrind/valgrind-test.sh');
-    assert_script_run 'bash -e valgrind-test.sh';
+    assert_script_run 'bash -e -x valgrind-test.sh';
```

Comment:
Can we modify the `valgrind-test.sh` script and add `-v` option for valgrind command?

Reply:
```suggestion
    assert_script_run 'bash -ex valgrind-test.sh';
```